LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -197,10 +197,17 @@ def parse_config(data, me_url, mdsd_url, is_lad, az_resource_id, subscription_id
             namespace = MetricsExtensionNamepsace
             if is_vmi or is_vmi_rate_counter:
                 namespace = "insights.virtualmachine"
+
+            if is_vmi_rate_counter:
+                # Adding "_rated" as a substring for vmi rate metrics to avoid renaming collisions
+                plugin_name = plugin + "_rated"
```

Comment:
Is that something that's expected / parsed OK on receiving end?

Reply:
Yes, the mdmratemetrics plugin for telegraf adds the "_rated" at the end of every field it aggregates. This was done so that later when we rename, it only renames the rate fields and not others. (there were renaming collisions with the other fields like memory percentage, processor utilization etc. that were getting lost due to all VMI metrics being renamed under the one measurement name - insights.virtualmachine